subscriptions/views.py

```diff
@@ -66,6 +66,28 @@ def post(self, request, *args, **kwargs):
         return Response(accepted, status=status)
 
 
+class SubscriptionResend(APIView):
+
+    """ Triggers a re-send for the current subscription message
+    """
+    permission_classes = (IsAuthenticated,)
+
+    def post(self, request, *args, **kwargs):
+        """ Validates subscription data before creating Outbound message
+        """
+        # Look up subscriber
+        subscription_id = kwargs["subscription_id"]
+        if Subscription.objects.filter(id=subscription_id).exists():
+            status = 201
+            accepted = {"accepted": True}
+            store_resend_request.apply_async(args=[subscription_id])
+        else:
+            status = 400
+            accepted = {"accepted": False,
+                        "reason": "Missing subscription in control"}
```

Comment:
I'm not sure that this error message is descriptive enough, I'm not quite sure what "control" is here. Maybe something like `Cannot find subscription with ID {subscription_id}`